### DISCUSSION TITLE
Any ideas for adding Column and Row Spanning for the Table header as well as Cells?

### DISCUSSION THREAD

**Discussion by Sajith15** [KevinVandy/material-react-table#239] [Ideas]
Any ideas for adding Column and Row Spanning for the Table header as well as Cells? This would be a really great addition and put this package far ahead of the alternatives!

**Comment by Sajith15** [comment]


**Reply by KevinVandy** [reply]
Sorry, but I don't think this library will be able to support custom spanning like that.
